src/layouts/MenuLayout/components/MenuPanel/tests/MenuPanel.test.js

```diff
@@ -0,0 +1,298 @@
+// @flow strict
+
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import {
+  divDecimals,
+  formatBalance,
+} from 'utils/numbers'
+
+jest.mock('../../../../../utils/sprite/spriteUI', () => ({
```

Comment:
It is possible to mock this before all unit tests
https://github.com/jibrelnetwork/jwallet-web/blob/380e03a7537484f9d7dc4a4898b24c21f25fa7c8/.jest/register-context.js
https://github.com/jibrelnetwork/jwallet-web/blob/380e03a7537484f9d7dc4a4898b24c21f25fa7c8/jest.config.js#L36

It's done in my PR